src/main/java/io/camunda/testing/assertions/ProcessInstanceAssertions.java

```diff
@@ -0,0 +1,413 @@
+package io.camunda.testing.assertions;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.camunda.testing.filters.StreamFilter;
+import io.camunda.zeebe.client.api.response.ProcessInstanceEvent;
+import io.camunda.zeebe.protocol.record.Record;
+import io.camunda.zeebe.protocol.record.intent.Intent;
+import io.camunda.zeebe.protocol.record.intent.ProcessInstanceIntent;
+import io.camunda.zeebe.protocol.record.intent.ProcessMessageSubscriptionIntent;
+import io.camunda.zeebe.protocol.record.value.BpmnElementType;
+import io.camunda.zeebe.protocol.record.value.ProcessInstanceRecordValue;
+import io.camunda.zeebe.protocol.record.value.ProcessMessageSubscriptionRecordValue;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import java.util.Optional;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+import org.assertj.core.api.AbstractAssert;
+import org.camunda.community.eze.RecordStreamSource;
+
+public class ProcessInstanceAssertions
+    extends AbstractAssert<ProcessInstanceAssertions, ProcessInstanceEvent> {
+
+  private RecordStreamSource recordStreamSource;
+
+  public ProcessInstanceAssertions(
+      final ProcessInstanceEvent actual, final RecordStreamSource recordStreamSource) {
+    super(actual, ProcessInstanceAssertions.class);
+    this.recordStreamSource = recordStreamSource;
+  }
+
+  /**
+   * Verifies the expectation that the process instance is started. This will also be true when the
+   * process has been completed or terminated.
+   *
+   * @return this {@link ProcessInstanceAssertions}
+   */
+  public ProcessInstanceAssertions isStarted() {
+    final Optional<Record<ProcessInstanceRecordValue>> processInstanceRecord =
```

Comment:
:x: I think this condition is incorrect. I think you need to look for an ACTIVATED intent.

For example, when you start a process instance and immediately an incident is created, or the command gets rejected, then this method would return true, but the process instance was never started.